### ISSUE TITLE
Missing Cloudwatch Insights Report query support

### ISSUE THREAD

**Issue by mourya92** [grafana/grafana#60345]
**What happened**:
Creating Github issue for this : https://github.com/grafana/grafana/discussions/60318

TLDR: Add support to view graphs of report data and list of contributors found by Cloudwatch Contributor Insight rules using GetInsightRuleReport API

**What you expected to happen**:
Grafana gives capability to : 
- List Cloudwatch Contributor Insight rules
- Query specific Contributor Insight rule Report by taking in Start Time, End Time,  MaxContributorCount,  Period, Rule Name and get a Timeseries data in response for visualization.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:
Today Grafana supports `INSIGHT_RULE_METRIC` support via GetMetricData API. 
We cannot query insights report data using Cloudwatch GetMetricData API. GetMetricData API can only be used to get stats of Insights report data but not the actual report itself.

**Environment**: 
- Grafana version: 8.5.11
- Data source type & version: CloudWatch
- OS Grafana is installed on: Amazon Linux 2
- User OS & Browser: Chrome/Firefox
- Grafana plugins: CloudWatch, TimeSeries
- Others:

**Comment by kevinwcyu** [comment]
Hello, for this feature request we would like to gather more information on how many people would want it and other use cases the community may have. We are going to close it as a ticket and leave it as a [github discussion](https://github.com/grafana/grafana/discussions/60318) and the team can see what feedback we get and prioritize it with our other commitments when we plan our work. Thank you!
